<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Adobe Devanagari</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>天然气产气量预测系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="import_data">
       <attribute name="title">
        <string>模型训练</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="text_train_path">
              <property name="font">
               <font>
                <family>宋体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string>请选择训练数据文件所在目录</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_import_train">
              <property name="font">
               <font>
                <family>宋体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>导入文件</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_start_train">
              <property name="font">
               <font>
                <family>宋体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>开始训练</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_clear_data">
              <property name="text">
               <string>清除数据</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QTextBrowser" name="textBrowser"/>
              </item>
              <item>
               <widget class="QProgressBar" name="progressBar">
                <property name="value">
                 <number>24</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>参数选择</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>预测天数</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="spinBox"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>天</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ap_analyse">
       <attribute name="title">
        <string>聚类分析</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5">
          <item>
           <widget class="QListWidget" name="leftlist">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stack">
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="model_prediction">
       <attribute name="title">
        <string>产气量预测</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLineEdit" name="text_model_path">
                <property name="placeholderText">
                 <string>请选择模型所在文件夹</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_import_model">
                <property name="text">
                 <string>导入预测模型</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLineEdit" name="text_dict_path">
                <property name="placeholderText">
                 <string>请选择聚类数据</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_import_dict">
                <property name="text">
                 <string>导入聚类字典</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
              <item>
               <widget class="QLineEdit" name="text_predict_path">
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="placeholderText">
                 <string>请选择预测数据文件所在文件夹</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_import_predata">
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>导入预测文件</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPushButton" name="btn_start_predict">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>宋体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>开始预测</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_clear_predata">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>清除数据</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,8">
            <item>
             <widget class="QListWidget" name="listWidget"/>
            </item>
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page"/>
              <widget class="QWidget" name="page_2"/>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar_2">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>导出</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>图片</string>
     </property>
     <addaction name="action_export_scatter"/>
     <addaction name="action_export_line"/>
     <addaction name="action_export_predict_png"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="action_export_predict_data"/>
    <addaction name="action_export_model"/>
    <addaction name="action_export_dict"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_version"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action123">
   <property name="text">
    <string>123</string>
   </property>
  </action>
  <action name="actionshiyongshuoming">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="action_export_scatter">
   <property name="text">
    <string>聚类散点图</string>
   </property>
  </action>
  <action name="action_export_line">
   <property name="text">
    <string>聚类折线图</string>
   </property>
  </action>
  <action name="action_export_predict_png">
   <property name="text">
    <string>预测折线图</string>
   </property>
  </action>
  <action name="action_export_predict_data">
   <property name="text">
    <string>预测数据</string>
   </property>
  </action>
  <action name="action_version">
   <property name="text">
    <string>版本号</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_export_model">
   <property name="text">
    <string>预测模型</string>
   </property>
  </action>
  <action name="action_export_dict">
   <property name="text">
    <string>聚类字典</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
